docs/installation/introduction.rst

```diff
@@ -119,32 +247,59 @@ other ranges during the
 :ref:`Bootstrap configuration <Bootstrap Configuration>`.
 
 
-.. _installation-intro-install-plan:
-
-Installation Plan
-^^^^^^^^^^^^^^^^^
-In this guide, the depicted installation procedure is for a medium-sized
-cluster, using three control plane nodes and two worker nodes. Refer to
-the :doc:`/installation/index` for extensive explanations of possible
-cluster architectures.
-
-.. note::
-
-   This image depicts the architecture deployed with this installation guide.
-
-   .. image:: img/architecture.png
-      :width: 100%
-
-   .. todo::
-
-      - describe architecture schema, include legend
-      - improve architecture explanation and presentation
-
-The installation process can be broken down into the following steps:
-
-#. :doc:`Setup <./setup>` of the environment (with requirements and example
-   OpenStack deployment)
-#. :doc:`Deployment <./bootstrap>` of the :term:`Bootstrap node`
-#. :doc:`Expansion <./expansion>` of the cluster from the Bootstrap node
-
-.. todo:: Include a link to example Solution deployment?
+Additional Notes
+^^^^^^^^^^^^^^^^
+
+.. _installation-intro-sizing:
+
+Sizing
+""""""
+Defining an appropriate sizing for the machines in a MetalK8s cluster strongly
+depends on the selected architecture and the expected future variations to
+this architecture. Refer to the documentation of the applications planned to
+run in the deployed cluster before completing the sizing, as their needs will
+compete with the cluster's.
+
+Each :ref:`role<node-roles>`, describing a group of services, requires a
+certain amount of resources for it to run properly. If multiple roles are used
+on a single Node, these requirements add up:
+
+- Bootstrap services, including Salt Master and its API, package repositories,
+  and container registries, need a minimum of 1 CPU core and 2 GB of RAM
+- Control plane services, including Kubernetes API and its backing ``etcd``
+  database, require at least 0.5 CPU core and 1.5 GB of RAM
+- Infra services, running Prometheus and Alertmanager among others, need 0.5
+  CPU core and 1.5 GB of RAM
+
+These numbers are not accounting for highly unstable workloads or other sources
+of unpredictable load on the cluster services, and it is recommended to provide
+an additional 50% of resources as a safety margin.
+
+Each machine in the cluster should have a root partition of at least 40 GB.
+An extra partition for ``etcd`` should be provisioned on control plane Nodes
+(see :ref:`this note<Setup etcd partition>` for more details). Prometheus and
+Alertmanager also require storage, as explained in
+:ref:`this section<Provision Prometheus Storage>`.
+
+.. _installation-intro-cloud:
+
+Deploying with Cloud Providers
+""""""""""""""""""""""""""""""
+Installing MetalK8s on virtual machines in cloud environments is perfectly
+achievable. Note however that most cloud providers have their own offerings
+for hosted Kubernetes clusters, which can save time and efforts (operation
+of the cluster being delegated).
```

Comment:
I would remove this sentence as the one reading this doc wants to install MetalK8s and probably made his choice between hosted K8s services and MetalK8s

Reply:
OK, will remove.